src/ui/Facet/Facet.ts

```diff
@@ -275,8 +277,16 @@ export class Facet extends Component {
      * option, or `occurrences` if no sort criteria is specified.
      */
     sortCriteria: ComponentOptions.buildStringOption({
-      postProcessing: (value, options: IFacetOptions) =>
-        value || (options.availableSorts.length > 0 ? options.availableSorts[0] : 'occurrences'),
+      postProcessing: (value, options: IFacetOptions) => {
+        const logger = new Logger(Facet);
+        const valueIfUndefined = value || (options.availableSorts.length > 0 ? options.availableSorts[0] : 'occurrences');
+        if (valueIfUndefined && QueryUtils.isValidSortCriteria(valueIfUndefined)) {
+          return valueIfUndefined;
+        } else {
+          logger.warn(`The Facet component doesn't accept ${valueIfUndefined} as the value for the sortCriteria option.`);
```

Comment:
Here in the warning message, it should log the available/valid sort criteria option.

Reply:
Will do